packages/manager/modules/telecom-spare/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@ovh-ux/manager-telecom-spare",
+  "version": "0.0.0",
```

Comment:
If you're not using this module outside the `manager` repository, you can switch it to `private`